lib/resources/postgres_session.rb

```diff
@@ -39,25 +41,32 @@ class PostgresSession < Inspec.resource(1)
         its('output') { should eq('') }
       end
     "
+    attr_reader :user, :pass, :host, :db, :escaped_query, :psql_cmd
 
     def initialize(user, pass, host = nil)
       @user = user || 'postgres'
       @pass = pass
       @host = host || 'localhost'
     end
 
-    def query(query, db = [])
+    def escaped_query(query)
```

Comment:
This should be a private method, moved toward the bottom of this file. Users shouldn't be able to call this directly.